change/@fluentui-react-swatch-picker-preview-3c1d6888-87ae-4c23-8e52-31fd298123de.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31358&commitId=754881177459250613776e31fc957306ec48652a&env=prod&clientType=FLUENTUI)


  <details><summary><b>SwatchPicker Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.default - Dark Mode.default.chromium.png|148| Changed|
|SwatchPicker Converged.default - High Contrast.default.chromium.png|171| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>